### PR TITLE
Chapter_8

### PR DESCRIPTION
**Pull request by hpt1512**
1. Link excel: https://docs.google.com/spreadsheets/d/1EYHDUr9EZArzCBvuTu1KBFbGnDhHCCfbDMHw0CzBH8g/edit?usp=sharing
2. Rubocop
image

### CODE REVIEW

**Comment by hpt1512** [comment]
Ready

File: app/views/sessions/create.html.erb [hunk 1]
```diff
@@ -0,0 +1,2 @@
+<h1>Sessions#create</h1>
```
Comment (vanvtt-0952) [inline app/views/sessions/create.html.erb]: Xóa file

File: app/views/sessions/destroy.html.erb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<h1>Sessions#destroy</h1>
+<p>Find me in app/views/sessions/destroy.html.erb</p>
```
Comment (vanvtt-0952) [inline app/views/sessions/destroy.html.erb]: Xoá file

File: app/views/sessions/new.html.erb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+<% provide :title, t("log_in") %>
+<h1><%= t("log_in")%></h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for :session, url: login_path do |f| %>
+    <%= f.label :email %>
+    <%= f.email_field :email, class: "form-control" %>
+    <%= f.label :password %>
+    <%= f.password_field :password, class: "form-control" %>
+    <%= f.submit "Log in", class: "btn btn-primary" %>
```
Comment (vanvtt-0952) [inline app/views/sessions/new.html.erb]: I18n

File: config/routes.rb [hunk 4]
```diff
@@ -1,10 +1,13 @@
 Rails.application.routes.draw do
+  get "/login", to: "sessions#new"
+  post "/login", to: "sessions#create"
+  delete "/logout", to: "sessions#destroy"
```
Comment (vanvtt-0952) [inline config/routes.rb:6]: Sau line root nha

**Comment by vanvtt-0952** [comment]
Merged
Comment nhỏ fix oy mợt

**Pull request merged**